src/EditorFeatures/Text/Microsoft.CodeAnalysis.EditorFeatures.Text.csproj

```diff
@@ -35,21 +35,21 @@
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.CSharp.InteractiveEditorFeatures" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.InteractiveEditorFeatures" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.VisualBasic.InteractiveEditorFeatures" />
-    <InternalsVisibleToTypeScript Include="Microsoft.CodeAnalysis.TypeScript.EditorFeatures" />
-    <InternalsVisibleToTypeScript Include="Microsoft.VisualStudio.LanguageServices.TypeScript" />
-    <InternalsVisibleToRemoteLS Include="Microsoft.VisualStudio.LanguageServices.Remote" />
-    <InternalsVisibleToRemoteLS Include="Microsoft.VisualStudio.LanguageServices.Remote.CSharp" />
-    <InternalsVisibleToFSharp Include="FSharp.Editor" />
-    <InternalsVisibleToFSharp Include="FSharp.LanguageService" />
+    <InternalsVisibleTo Include="Microsoft.CodeAnalysis.TypeScript.EditorFeatures" Key="$(TypeScriptKey)" />
```

Comment:
:memo: I would prefer to keep the original form here, and use a target somewhere to convert `InternalsVisibleToTypeScript` items to the appropriate `InternalsVisibleTo` items.

Reply:
It's too complicated (not sure if possible) to write such target.

Reply:
Other repos have just a single syntax for `InternalsVisibleTo` and have not had issues with it. 
Perhaps Roslyn uses IVTs more than other repos, but I'd still prefer we have the same syntax across repos. Ultimately, we should only have IVTs to external assemblies in one assembly that defines internal interfaces that we provide to our internal partners. I believe we already do so for Razor.

Reply:
i don't see a substantial change in readibility here. The name is still clearly visible, it's just now in an attribute name vs. element name.

Reply:
@sharwell Are you satisfied with the reasoning?